### PR TITLE
Generate assertion for Object Type Application

### LINKED ISSUE #16: Generate assertion for Object Type Application?
https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#type-application describes possible Type Application for Object: `Object<string, someType>`. It's easy to implement assert generation for this case as

```js
/**
 * @param {Object<string, number>} x
 */
```

=>

```js
Object.values(x).every(function (item) {
    return typeof item  === 'number';
});
```

AFAIK there is no need to generate assertion for first type (for object keys), as it will always be `string`.

If it looks good, then I'll make a PR.

### PR DESCRIPTION
**Pull request by victor-homyakov**
Generate assertion for `@param Object<string, someType>` to check that all object values are of `someType`

close #16

### CODE REVIEW

File: src/AssertGenerator.js [hunk 1]
```diff
@@ -88,29 +88,30 @@ export default class AssertGenerator {
...
+            case "UnionType":
+                return UnionType(node, generator);
+            case "TypeApplication":
+                return TypeApplication(node, generator);
+            case "RestType":
+                return RestType(node, generator);
+            case "NullableType":
+                return NullableType(node, generator);
+            case "NonNullableType":
+                return NonNullableType(node, generator);
```
Comment (azu) [inline src/AssertGenerator.js:109]: Can you add `default: return;` case?

Comment (victor-homyakov) [reply src/AssertGenerator.js:109]: Sure

Comment (victor-homyakov) [reply src/AssertGenerator.js:109]: Done

**Comment by azu** [comment]
Ok. I'll merge it soon.

**Comment by azu** [comment]
Release https://github.com/azu/jsdoc-to-assert/releases/tag/2.8.0

**Pull request merged**